<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgProgress</class>
 <widget class="QDialog" name="CQDlgProgress">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>75</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Progress</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QLabel" name="qqLine1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>481</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qqLine2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>481</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="qqLine3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>481</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
